docs/reference/components/sources/kubernetes_logs.cue

```diff
@@ -0,0 +1,127 @@
+package metadata
+
+components: sources: kubernetes_logs: {
+	title: "Kubernetes Logs"
+
+	classes: {
+		commonly_used: true
+		delivery:      "best_effort"
+		deployment_roles: ["daemon"]
+		development:   "stable"
+		egress_method: "stream"
+	}
+
+	features: {
+		collect: {
+			checkpoint: enabled: false
+		}
+		multiline: enabled: false
+	}
+
+	support: {
+		platforms: {
+			"aarch64-unknown-linux-gnu":  true
+			"aarch64-unknown-linux-musl": true
+			"x86_64-apple-darwin":        true
+			"x86_64-pc-windows-msv":      true
+			"x86_64-unknown-linux-gnu":   true
+			"x86_64-unknown-linux-musl":  true
+		}
+
+		requirements: [
+			"""
+				[Kubernetes][urls.kubernetes] version `>= 1.14` is required.
+				""",
+		]
+		warnings: []
+		notices: []
+	}
+
+	configuration: {
+		annotation_fields: {
+			common:      false
+			description: "Configuration for how the events are annotated with Pod metadata."
+			required:    false
+			type: object: {
+				examples: []
+				options: {
```

Comment:
@MOZGIII can you make sure the options here are still accurate given the recent changes?